### ISSUE TITLE
how to add gan loss in this code

### ISSUE THREAD

**Issue by Lvhhhh** [NJU-Jet/SR_Mobile_Quantization#12]
how to add gan loss to this code? this results  look so blur

**Comment by NJU-Jet** [comment]
I think there is something wrong. I checked the predicted DIV2K validation results and they all look fine. Do you use your own images or DIV2K validation set?

**Comment by NJU-Jet** [comment]
As for gan loss, please refer to https://github.com/sanghyun-son/EDSR-PyTorch
